From 6cca2462d98c1da580faeaae5e33c99daf1a6c82 Mon Sep 17 00:00:00 2001
From: Adrien Destugues <pulkomandy@gmail.com>
Date: Fri, 7 Nov 2014 09:42:21 +0100
Subject: libGL: link against libbe

* This uses be_bold_font, and some apps (Pixel and other BeOS stuff)
would fail to find the symbol otherwise.

diff --git a/src/hgl/libgl-haiku/Makefile b/src/hgl/libgl-haiku/Makefile
index 9d7d301..8195813 100644
--- a/src/hgl/libgl-haiku/Makefile
+++ b/src/hgl/libgl-haiku/Makefile
@@ -43,7 +43,7 @@ $(GLAPI_LIB):
 
 # Make libGL
 $(TOP)/$(LIB_DIR)/$(GL_LIB_NAME): $(OBJECTS) $(GLAPI_LIB) Makefile
-	$(MKLIB) -o $(GL_LIB) -linker '$(CXX)' -ldflags '$(LDFLAGS)' \
+	$(MKLIB) -o $(GL_LIB) -linker '$(CXX)' -ldflags '$(LDFLAGS) -lbe' \
 		-cplusplus $(MKLIB_OPTIONS) -install $(TOP)/$(LIB_DIR) \
 		$(GL_LIB_DEPS) $(GLAPI_LIB) $(OBJECTS) $(GLU_LIB)
 
-- 
1.8.3.4

